wolfcrypt/src/hc128.c

```diff
@@ -271,14 +271,17 @@ static void Hc128_SetIV(HC128* ctx, const byte* inIv)
     for (i = 0; i < 64; i++)  setup_update(ctx);
 }
 
-
+#define HC128_KEY_NUMBYTES (128 >> 5)
 static WC_INLINE int DoKey(HC128* ctx, const byte* key, const byte* iv)
 {
   word32 i;
+  word32 keyc[HC128_KEY_NUMBYTES];
+
+  XMEMCPY(keyc, key, sizeof(keyc));
```

Comment:
`memcpy` wrap in `XSTREAM_ALIGN`?

Reply:
Alignment should have been fixed in wc_Hc128_SetKey().